index.js

```diff
@@ -1,52 +1,111 @@
 const colors = [ 'red' , 'white', 'black' , 'green' , 'orange'];
+--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- */
 
+// assign the button element to 'removeColorButton', and call 'click function it to  assign "removeColorButton".
+const removeColorButton = document.getElementsByTagName('button')[0];
+removeColorButton.addEventListener('click', ()  => 
+    {
+        //Assign mySelect element to select variable, use it to access the index and assing it to value "variable",
+        // then delete the selected index value
+        var select = document.getElementById('mySelect');
+        value = select.selectedIndex;
+        select.removeChild(select[value]);
+    });  
 
 
-
-//resolve // QUESTION 2 here
-
-
-
-
-
-
-//------------------------Question 3 ---------------------------
+/*------------------------------------------------Question 3 --------------------------------------------------------------------------------------------------------------------------
 //Now lets mix some maths and programming. In the function below (calculate_sphere) write a JavaScript program to calculate the volume and surface area of a sphere.
-//take the radius from the input value element with the 'id=radius' in the index.html file
-//And display the results in the elements with the 'id = volume' and 'id = area' respectively
+take the radius from the input value element with the 'id=radius' in the index.html file And display the results in the elements with the 'id = volume' and 'id = area' respectively
 //hint: the volumen of a sphere is ((4/3) × π × r^3) and the surface area is (4 × π × r^2)
-
-
-const calculate_sphere = () =>{
-
- }
-
-window.onload = document.getElementById('MyForm').onsubmit = calculate_sphere; // this execute the volume_sphere function each time the calculate (submit) button is clicked
-
-
-
-
-//------------------------Question 4 ---------------------------
-//Now in this Exercise we want you to create 3 buttons wich clicks actions are going to hide the respective question above with their aswer and if they are hidden then the click option should be display the question with their respective aswer
-// append the 3 buttons to the element with the 'id = a-4' as childrens
-
-
-//resolve // QUESTION 4 here
+-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- */
+const  calculate_sphere = () =>
+ {
+  // declare volume and surface vaiables
+  var volume;
+  var surface;
+  //get radius element and value from html document and assign it to radius variable, get the absolute value for the value.
+  var radius = document.getElementById('radius').value;
+  radius = Math.abs(radius);
```

Comment:
This to account for negative numbers, smart thinking looking at bad info a user could put in